app-web/halloy/autobuild/beyond

```diff
@@ -0,0 +1,7 @@
+abinfo "Installing desktop entry ..."
+install -Dm644 "$SRCDIR/assets/linux/org.squidowl.halloy.appdata.xml" -t "$PKGDIR/usr/share/appdata"
```

Comment:
All commands here should have their verbose flags enabled (`-v` for all instances here).